src/Server.jl

```diff
@@ -736,13 +736,24 @@ function make_pull_request(pp::ProcessedParams, rp::RequestParams, rbrn::RegBran
     pr = nothing
     repo = join(split(target_registry["repo"], "/")[end-1:end], "/")
     msg = ""
+    auth = get_user_auth()
     try
-        pr = create_pull_request(repo; auth=get_user_auth(), params=params)
+        pr = create_pull_request(repo; auth=auth, params=params)
         msg = "created"
         @debug("Pull request created")
     catch ex
         if is_pr_exists_exception(ex)
-            @debug("Pull request already exists, not creating")
+            @debug("Pull request already exists, trying to update body")
+            params = Dict(
+                "base" => target_registry["base_branch"],
+                "head" => split(repo, "/")[1] * ":" * brn,
+            )
+            prs = pull_request(repo; auth=auth, params=params)
+            if !isempty(prs)
+                pr = prs[1]
+                params = Dict("body" => params["body"])
```

Comment:
You have overwritten `params` here on L747 so the body does not exist, right?

Reply:
Right, that's true